crypto/s2n_hkdf.c

```diff
@@ -126,7 +126,11 @@ static int s2n_libcrypto_hkdf_extract(struct s2n_hmac_state *hmac, s2n_hmac_algo
     const EVP_MD *digest = NULL;
     POSIX_GUARD_RESULT(s2n_hmac_md_from_alg(alg, &digest));
 
-    size_t pseudo_rand_key_len = (size_t) pseudo_rand_key->size;
+    size_t hmac_size = EVP_MD_size(digest);
```

Comment:
Our impl `s2n_hash_digest_size` should return the same result right? Having the same check/code seems nicer since its easier to reason about.

Reply:
continue conversation in thread above